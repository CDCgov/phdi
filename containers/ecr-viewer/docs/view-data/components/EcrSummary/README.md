[**ecr-viewer**](../../../README.md) â€¢ **Docs**

***

[ecr-viewer](../../../README.md) / view-data/components/EcrSummary

# view-data/components/EcrSummary

## Index

### Functions

- [default](functions/default.md)
