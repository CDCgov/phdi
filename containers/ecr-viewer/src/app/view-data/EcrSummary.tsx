import {evaluate} from "fhirpath";
import {Bundle} from "fhir/r4";

interface EcrViewerProps {
    fhirPathMappings: PathMappings
    fhirBundle: Bundle
}

interface PathMappings {
    [key: string]: string;
}

const patientAddress = (fhirBundle: Bundle, fhirPathMappings: PathMappings) => {
    const streetAddresses = evaluate(fhirBundle, fhirPathMappings.patientStreetAddress).join("\n");
    const city = evaluate(fhirBundle, fhirPathMappings.patientCity);
    const state = evaluate(fhirBundle, fhirPathMappings.patientState);
    const zipCode = evaluate(fhirBundle, fhirPathMappings.patientZipCode);
    const country = evaluate(fhirBundle, fhirPathMappings.patientCountry);
    return(
    `${streetAddresses}
    ${city}, ${state}
    ${zipCode}${country && `, ${country}`}`);
}

const patientName = (fhirBundle: Bundle, fhirPathMappings: PathMappings) => {
    const givenNames = evaluate(fhirBundle, fhirPathMappings.patientGivenName).join(" ");
    const familyName = evaluate(fhirBundle, fhirPathMappings.patientFamilyName);

    return `${givenNames} ${familyName}`;
}

const patientContactInfo = (fhirBundle: Bundle, fhirPathMappings: PathMappings) => {
    const phoneNumbers = evaluate(fhirBundle, fhirPathMappings.patientPhoneNumbers).map(phoneNumber => `${phoneNumber?.use?.charAt(0).toUpperCase() + phoneNumber?.use?.substring(1)} ${phoneNumber.value}`).join("\n");
    const emails = evaluate(fhirBundle, fhirPathMappings.patientEmails).map(email => `${email.value}`).join("\n");

    return `${phoneNumbers}
    ${emails}`
}

const EcrSummary = (
    {fhirPathMappings, fhirBundle}: EcrViewerProps
) => {
    const rr = evaluate(fhirBundle, fhirPathMappings.rrSection)[0];
    return (
        <div className={"info-container"}>
            <div
                className="usa-summary-box padding-3"
                aria-labelledby="summary-box-key-information"
            >
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Patient
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Name</h4></div>
                            <div className="grid-col-auto">
                                {patientName(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>DOB</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientDOB)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Address</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {patientAddress(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Contact</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {patientContactInfo(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                    </div>
                </div>
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Condition
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Reportable Condition</h4></div>
                            <div className="grid-col-fill">
                                {rr.entry[0].display.split("-")[1]}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>RCKMS Trigger Summary</h4></div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientDOB)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                                <h4>Lab results relevant to reportable condition</h4>
                            <div className="usa-card__container margin-left-0 margin-right-0 margin-top-1">
                                <div className="usa-card__header padding-top-2 padding-bottom-2 padding-left-3">
                                    <p>
                                        Hepatitis C IgG w/Rfx PCR (temp)<span
                                        className="usa-tag margin-left-1 bg-error-dark text-white">Abnormal</span>
                                    </p>
                                </div>
                                <div className={"card__line "}/>
                                <div className="usa-card__body padding-0">
                                    <table className="usa-table usa-table--borderless ecrTable">
                                        <thead>
                                        <tr>
                                            <th scope="col">Component</th>
                                            <th scope="col">Value</th>
                                            <th scope="col">Ref Range</th>
                                            <th scope="col">Specimen</th>
                                        </tr>
                                        </thead>
                                        <tbody>
                                        <tr>
                                            <td>Hepatitis C Antibody (temp)</td>
                                            <td>Positive (A) (temp)</td>
                                            <td>Not Detected (temp)</td>
                                            <td>Blood (temp)</td>
                                        </tr>
                                        </tbody>
                                    </table>
                                    <div className={"card__line"}/>
                                    {/*todo: make a class for these*/}
                                    <div
                                        className={"padding-left-3 padding-right-3 padding-top-205 padding-bottom-205"}>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Analysis time</h4></div>
                                            <div className="grid-col-auto text-pre-line">
                                                N/A (temp)
                                            </div>
                                        </div>
                                        {/*todo: replace with grey line*/}
                                        <div className={"section__line"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Collection time</h4></div>
                                            <div className="grid-col-auto text-pre-line">
                                                05/12/2022 6:00 AM CDT (temp)
                                            </div>
                                        </div>
                                        {/*todo: replace with grey line*/}
                                        <div className={"section__line"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Received time</h4></div>
                                            <div className="grid-col-auto text-pre-line">
                                                05/12/2022 11:30 AM CDT (temp)
                                            </div>
                                        </div>
                                        {/*todo: replace with grey line*/}
                                        <div className={"section__line"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Notes</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                A detected result is positive and indicates the presence of the virus in the sample. A not detected result indicates that the test did not detect the virus in the sample. This test was performed using a multiplexed nucleic acid amplification test that is labeled Emergence Use Only by the U.S. FDA. Results of this test should not be used as the sole basis for diagnosis, treatment, or other management decisions. (temp)
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <h4>Clinical sections relevant to reportable condition</h4>
                            <div className={"padding-top-05"}>No matching clinical data found in this eCR (temp) </div>
                        </div>
                    </div>
                </div>

            </div>
        </div>);
};

export default EcrSummary;