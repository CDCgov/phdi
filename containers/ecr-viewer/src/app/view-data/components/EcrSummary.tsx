import {evaluate} from "fhirpath";
import {Bundle} from "fhir/r4";
import {PathMappings, formatPatientName, formatPatientAddress, formatPatientContactInfo} from "../../utils";

interface EcrViewerProps {
    fhirPathMappings: PathMappings
    fhirBundle: Bundle
}


const EcrSummary = (
    {fhirPathMappings, fhirBundle}: EcrViewerProps
) => {
    return (
        <div className={"info-container"}>
            <div
                className="usa-summary-box padding-3"
                aria-labelledby="summary-box-key-information"
            >
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Patient
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Name</h4></div>
                            <div className="grid-col-fill">
                                {formatPatientName(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>DOB</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientDOB)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Address</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {formatPatientAddress(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>Patient Contact</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {formatPatientContactInfo(fhirBundle, fhirPathMappings)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                    </div>
                </div>
                <div className="usa-summary-box__body">
                    <h3 id="summary-box-key-information">
                        About the Condition
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="data-title"><h4>Reportable Condition</h4></div>
                            <div className="grid-col-fill">
                                {evaluate(fhirBundle, fhirPathMappings.rrDisplayName)[0].split('-')[1]}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <div className="data-title"><h4>RCKMS Trigger Summary</h4></div>
                            <div className="grid-col-fill text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.rckmsTriggerSummary)}
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                                <h4>Lab results relevant to reportable condition</h4>
                            <div className="usa-card__container margin-left-0 margin-right-0 margin-top-1">
                                <div className="usa-card__header padding-top-2 padding-bottom-2 padding-left-3">
                                    <p>
                                        Hepatitis C IgG w/Rfx PCR (temp)<span
                                        className="usa-tag margin-left-1 bg-error-dark text-white">Abnormal</span>
                                    </p>
                                </div>
                                <div className={"card__line "}/>
                                <div className="usa-card__body padding-0">
                                    <table className="usa-table usa-table--borderless ecrTable">
                                        <thead>
                                        <tr>
                                            <th scope="col">Component</th>
                                            <th scope="col">Value</th>
                                            <th scope="col">Ref Range</th>
                                            <th scope="col">Specimen</th>
                                        </tr>
                                        </thead>
                                        <tbody>
                                        <tr>
                                            <td>Hepatitis C Antibody (temp)</td>
                                            <td>Positive (A) (temp)</td>
                                            <td>Not Detected (temp)</td>
                                            <td>Blood (temp)</td>
                                        </tr>
                                        </tbody>
                                    </table>
                                    <div className={"card__line"}/>
                                    <div
                                        className={"padding-left-3 padding-right-3 padding-top-205 padding-bottom-205"}>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Analysis time</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                N/A (temp)
                                            </div>
                                        </div>
                                        <div className={"section__line_gray"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Collection time</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                05/12/2022 6:00 AM CDT (temp)
                                            </div>
                                        </div>
                                        <div className={"section__line_gray"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Received time</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                05/12/2022 11:30 AM CDT (temp)
                                            </div>
                                        </div>
                                        <div className={"section__line_gray"}/>
                                        <div className="grid-row">
                                            <div className="data-title"><h4>Notes</h4></div>
                                            <div className="grid-col-fill text-pre-line">
                                                A detected result is positive and indicates the presence of the virus in
                                                the sample. A not detected result indicates that the test did not detect
                                                the virus in the sample. This test was performed using a multiplexed
                                                nucleic acid amplification test that is labeled Emergence Use Only by
                                                the U.S. FDA. Results of this test should not be used as the sole basis
                                                for diagnosis, treatment, or other management decisions. (temp)
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                            <h4>Clinical sections relevant to reportable condition</h4>
                            <div className={"padding-top-05"}>No matching clinical data found in this eCR (temp) </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>);
};

export default EcrSummary;