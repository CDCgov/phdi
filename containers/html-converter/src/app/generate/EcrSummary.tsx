import {evaluate} from "fhirpath";
import fs from "fs";
import {parse} from "yaml";
import {Bundle} from "fhir/r4";

interface EcrViewerProps {
    fhirBundle: any
}

const EcrSummary = (
    // {fhirBundle}: EcrViewerProps
) => {
    const file = fs.readFileSync('./src/app/generate/fhirPath.yml', 'utf8').toString();
    const fhirBundle: Bundle = JSON.parse(fs.readFileSync('./src/app/generate/exampleBundle.json', 'utf8').toString());
    const fhirPathMappings = parse(file);
    return (
        <div>
            <h1>Quick eCR Summary</h1>
            <div
                className="usa-summary-box padding-3"
                aria-labelledby="summary-box-key-information"
            >
                <div className="usa-summary-box__body">
                    <h3
                        className="usa-summary-box__heading padding-top-105 padding-bottom-1"
                        id="summary-box-key-information"
                    >
                        About the Patient
                    </h3>
                    <div className="usa-summary-box__text">
                        <div className="grid-row">
                            <div className="grid-col-2 text-bold">Patient Name</div>
                            <div className="grid-col-auto">
                                {evaluate(fhirBundle, fhirPathMappings.patientGivenName).join(" ")} {evaluate(fhirBundle, fhirPathMappings.patientFamilyName)}
                            </div>
                        </div>
                        <div className={"section__line"} />
                        <div className="grid-row">
                            <div className="grid-col-2 text-bold">Patient Address</div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientStreetAddress).join("\n")} {'\n'}
                                {evaluate(fhirBundle, fhirPathMappings.patientCity)}, {evaluate(fhirBundle, fhirPathMappings.patientState)} {'\n'}
                                {evaluate(fhirBundle, fhirPathMappings.patientZipCode)}, USA
                            </div>
                        </div>
                        <div className={"section__line"}/>
                        <div className="grid-row">
                        <div className="grid-col-2 text-bold">Patient Contact</div>
                            <div className="grid-col-auto text-pre-line">
                                {evaluate(fhirBundle, fhirPathMappings.patientPhoneNumbers).map(phoneNumber => {
                                    return `tel: (${phoneNumber.use}) ${phoneNumber.value}`
                                }).join("\n")}
                                {'\n'}
                                {evaluate(fhirBundle, fhirPathMappings.patientEmails).map(email => {
                                    return `email: ${email.value}`
                                }).join("\n")}
                            </div>
                        </div>
                        <div className={"section__line"} />
                    </div>
                </div>
            </div>
        </div>);
};

export default EcrSummary;