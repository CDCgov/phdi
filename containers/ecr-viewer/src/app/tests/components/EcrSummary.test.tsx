import { render, screen } from "@testing-library/react";
import { axe } from "jest-axe";
import EcrSummary, {
  ConditionSummary,
} from "../../view-data/components/EcrSummary";

describe("EcrSummary", () => {
  const patientDetails = [
    {
      title: "Patient Name",
      value: "ABEL CASTILLO",
    },
    {
      title: "DOB",
      value: "04/15/2015",
    },
    {
      title: "Sex",
      value: "male"
    },
    {
      title: "Patient Address",
      value: "1050 CARPENTER ST\nEDWARDS, CA\n93523-2800, US",
    },
    {
      title: "Patient Contact",
      value: "Home (818)419-5968\nMELLY.C.A.16@GMAIL.COM",
    },
  ];
  const encounterDetails = [
    {
      title: "Facility Name",
      value: "PRM- Palmdale Regional Medical Center",
    },
    {
      title: "Facility Contact",
      value: "(661)382-5000",
    },
    {
      title: "Encounter Date/Time",
      value: "Start: 05/13/2022 7:25 AM UTC\nEnd: 05/13/2022 9:57 AM UTC",
    },
    {
      title: "Encounter Type",
      value: "Emergency",
    },
  ];
  const covidConditionDetails: ConditionSummary[] = [
    {
      title: "Influenza caused by Influenza A virus subtype H5N1 (disorder)",
      snomed: "test-snomed-123",
      conditionDetails: [
        {
          title: "RCKMS Rule Summary",
          value: "covid summary",
        },
      ],
      clinicalDetails: [
        {
          title: "Relevant Clinical",
          value: "covid clinical",
        },
      ],
      labDetails: [
        {
          title: "Relevant Labs",
          value: "covid lab",
        },
      ],
    },
  ];
  const hepConditionDetails: ConditionSummary[] = [
    {
      title: "Hep C",
      snomed: "test-snomed-456",
      conditionDetails: [
        {
          title: "RCKMS Rule Summary",
          value: "hep c summary",
        },
      ],
      clinicalDetails: [
        {
          title: "Relevant Clinical",
          value: "hep c clinical",
        },
      ],
      labDetails: [
        {
          title: "Relevant Labs",
          value: "hep c lab",
        },
      ],
    },
  ];

  beforeAll(() => {});
  it("should match snapshot", () => {
    const { container } = render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={covidConditionDetails}
      />,
    );

    expect(container).toMatchSnapshot();
  });
  it("should pass accessibility test", async () => {
    const { container } = render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={covidConditionDetails}
      />,
    );

    expect(await axe(container)).toHaveNoViolations();
  });
  it("should open the condition details when there is one", () => {
    render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={covidConditionDetails}
      />,
    );

    expect(screen.getByText("covid summary")).toBeVisible();
  });
  it("should open the condition when the snomed matches", () => {
    render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={[...covidConditionDetails, ...hepConditionDetails]}
        snomed={"test-snomed-456"}
      />,
    );

    expect(screen.getByText("hep c summary")).toBeVisible();
  });
  it("should open no condition details when there is many and no match", () => {
    render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={[...covidConditionDetails, ...hepConditionDetails]}
      />,
    );

    expect(screen.getByText("hep c summary")).not.toBeVisible();
    expect(screen.getByText("covid summary")).not.toBeVisible();
  });
  it("should show 0 reportable conditions tag", () => {
    render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={[]}
      />,
    );

    expect(screen.getByText("0 CONDITIONS FOUND"));
  });
  it("should show 1 reportable condition tag", () => {
    render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={covidConditionDetails}
      />,
    );

    expect(screen.getByText("1 CONDITION FOUND"));
  });
  it("should show 2 reportable conditions tag", () => {
    render(
      <EcrSummary
        patientDetails={patientDetails}
        encounterDetails={encounterDetails}
        conditionSummary={[...covidConditionDetails, ...hepConditionDetails]}
      />,
    );

    expect(screen.getByText("2 CONDITIONS FOUND"));
  });
});