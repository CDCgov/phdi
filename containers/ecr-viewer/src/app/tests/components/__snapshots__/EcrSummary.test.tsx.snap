// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EcrSummary should match snapshot 1`] = `
<div>
  <div
    class="info-container"
  >
    <div
      aria-labelledby="summary-box-key-information"
      class="usa-summary-box padding-3"
    >
      <div
        class="usa-summary-box__body margin-bottom-05"
      >
        <h2
          class="summary-box-key-information side-nav-ignore"
          id="about-the-patient"
        >
          About the Patient
        </h2>
        <div
          class="usa-summary-box__text"
        >
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Patient Name
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                ABEL CASTILLO
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                DOB
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                04/15/2015
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Patient Address
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                1050 CARPENTER ST
EDWARDS, CA
93523-2800, US
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Patient Contact
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                Home (818)419-5968
MELLY.C.A.16@GMAIL.COM
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
        </div>
      </div>
      <div
        class="usa-summary-box__body"
      >
        <h2
          class="summary-box-key-information side-nav-ignore"
          id="about-the-encounter"
        >
          About the Encounter
        </h2>
        <div
          class="usa-summary-box__text"
        >
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Facility Name
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                PRM- Palmdale Regional Medical Center
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Facility Address
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                38600 Medical Center Drive
Palmdale, CA
93551, USA
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Facility Contact
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                (661)382-5000
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Encounter Date/Time
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                Start: 05/13/2022 7:25 AM UTC
End: 05/13/2022 9:57 AM UTC
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Encounter Type
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                Emergency
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
        </div>
      </div>
      <div
        class="usa-summary-box__body"
      >
        <h2
          class="summary-box-key-information side-nav-ignore"
          id="about-the-condition"
        >
          About the Condition
        </h2>
        <div
          class="usa-summary-box__text"
        >
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                Reportable Condition
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                Influenza caused by Influenza A virus subtype H5N1 (disorder)
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div>
            <div
              class="grid-row"
            >
              <div
                class="data-title"
              >
                RCKMS Rule Summary
              </div>
              <div
                class="grid-col-auto maxw7 text-pre-line"
              >
                Cough
              </div>
            </div>
            <div
              class="section__line_gray"
            />
          </div>
          <div
            class="ecr-summary-title-long"
            id="relevant-labs"
          >
            Lab Results Relevant to Reportable Condition
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
